File: frontend/src/app/models/response/query/getTaskItemByIdQueryResponseModel.ts
export default class getTaskItemByIdQueryResponseModel {
    constructor(name: string, createDate: Date, isDone: boolean, description?: string) {
        this.name = name
        this.createDate = createDate
        this.isDone = isDone
        this.description = description
    }

    name: string
    createDate: Date
    isDone: boolean
    description?: string
}

File: TODOList.Domain/Models/ResponseModels/QueryResponseModels/GetTaskItemByIdQueryResponseModel.cs
using System;

namespace TODOList.Domain.Models.ResponseModels.QueryResponseModels
{
    public class GetTaskItemByIdQueryResponseModel
    {
        public GetTaskItemByIdQueryResponseModel(string name, string description, DateTime createDate, bool isDone)
        {
            Name = name;
            Description = description;
            CreateDate = createDate;
            IsDone = isDone;
        }

        public string Name { get; private set; }
        public string Description { get; private set; }
        public DateTime CreateDate { get; private set; }
        public bool IsDone { get; private set; }
    }
}
